.env.sample

```diff
@@ -30,6 +30,8 @@ PROFILE_IMAGES_BASE_URL=https://images.dev.terraso.org
 DATA_ENTRY_FILE_S3_BUCKET=files.dev.terraso.org
 DATA_ENTRY_FILE_BASE_URL=https://files.dev.terraso.org
 
+DB_BACKUP_S3_BUCKET=backup-bucket
```

Comment:
Suggestion: Can we have a `dev` bucket name named like the others, the have this structure: `files.dev.terraso.org`